### PR TITLE
docs: updated description, added example for external configuration…

### PR DESCRIPTION
**Pull request by tarang9211**
PR for Issue - https://github.com/webpack/webpack.js.org/issues/402

### CODE REVIEW

File: content/configuration/external-configs.md [hunk 1]
```diff
@@ -4,10 +4,36 @@ sort: 3
 contributors:
   - sokra
   - skipjack
+  - tarang9211
 ---
 
-...
+Webpack lets you define you configuration files in any language. The list of supported file extensions can be found at the [node-interpret](https://www.npmjs.com/package/interpret) package. Webpack will still interpret your file as pure JS!
```
Comment (simon04) [inline content/configuration/external-configs.md]: webpack is always lowercase, even at the beginning of a sentence.

File: content/configuration/external-configs.md [hunk 2]
```diff
@@ -4,10 +4,36 @@ sort: 3
...
 
-?> possible extensions, e.g. `.babel.js`
+For example if you use **coffeescript**, your file would be as follows:
 
-?> loader settings, e.g. `sassLoader`, `eslint` sections
\ No newline at end of file
+**webpack.config.coffee.js**
```
Comment (simon04) [inline content/configuration/external-configs.md]: Is it really `.coffee.js` or just `.coffee`? That's how I read the [extensions from interpret](https://www.npmjs.com/package/interpret#extensions)

File: content/configuration/external-configs.md (hunk 1 above)
Comment (simon04) [inline content/configuration/external-configs.md]: According to GitHub, the package is now in a repository called js-interpret: https://github.com/js-cli/js-interpret

Comment (simon04) [inline content/configuration/external-configs.md]: How is "Webpack will still interpret your file as pure JS" meant? interpret preprocesses/transpiles the file, and gives the result to webpack?

File: content/configuration/external-configs.md [hunk 3]
```diff
@@ -4,10 +4,36 @@ sort: 3
...
+
+```javascript
+HtmlWebpackPlugin = require('html-webpack-plugin')
+#installed via npm
+webpack = require('webpack')
+#to access built-in plugins
```
Comment (simon04) [inline content/configuration/external-configs.md]: I think, we can omit those two comments as they are used throughout the documentation.

File: content/configuration/external-configs.md (hunk 1 above)
Comment (tarang9211) [inline content/configuration/external-configs.md]: I'll change the link.

Comment (tarang9211) [inline content/configuration/external-configs.md]: I could be wrong about this! Maybe @TheLarkInn could help clarify this?

File: content/configuration/external-configs.md (hunk 2 above)
Comment (tarang9211) [inline content/configuration/external-configs.md]: I think it's `.coffee` Sorry about that

File: content/configuration/external-configs.md (hunk 1 above)
Comment (TheLarkInn) [inline content/configuration/external-configs.md]: Maybe better to say "webpack with the help of `node-interperet` will run your configuration through the language of choice" or something along these lines.

**Comment by tarang9211** [comment]
@simon04 just updated with your code-review suggestions! do you know any way to generate .yaml or typescript file from the js config file?

**Comment by simon04** [comment]
@tarang9211 Thank you. 

Might be best done manually. There doesn't seem to be any [webpack.config.yaml](https://github.com/search?utf8=%E2%9C%93&q=filename%3Awebpack.config.yaml&type=Code&ref=searchresults) around.

**Comment by tarang9211** [comment]
Alright, do you have any idea on where I could learn yaml or another example?

Since this is already merged, I'll submit another PR with those examples. Also, when this does patch go live on webpack.js.org ?

**Pull request merged**